react/Toggle/index.jsx

```diff
@@ -4,9 +4,9 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
 class Toggle extends Component {
-  onChange() {
+  onChange(e) {
     if (this.props.onToggle) {
-      this.props.onToggle(!this.props.checked)
+      this.props.onToggle(e.target.checked)
```

Comment:
Maybe it was used in another way by changing the props from the parent. I can't know for now.

Reply:
The usage as a controlled component is shown in the styleguide.

Reply:
Perfect so.